WordPress/Classes/ViewRelated/Post/PostEditor+MoreOptions.swift

```diff
@@ -64,9 +64,7 @@ extension PostEditor where Self: UIViewController {
         }
     }
 
-    private func displayPreviewNotAvialble() {
-        let title = NSLocalizedString("Preview Unavailable for Published Posts", comment: "Title on display preview error" )
-        let subtitle = NSLocalizedString("Update the published post to view your changes.", comment: "subtitle on display preview error" )
+    private func displayPreviewNotAvialble(title: String, subtitle: String? = nil) {
```

Comment:
I know this was already like that, but it the name says `Avialble` instead of `Available` 😬

Reply:
oops. Good catch.